styles/main.css

```diff
@@ -1,96 +1,103 @@
 body {
-    background-color: #62D8BB; 
-    font-family: 'Open Sans Condensed', sans-serif;  
-    }
-   
-   h1 {color: transparent; 

```

Comment:
I made some indentation edits in this update to bring your code to the left edge